### ISSUE TITLE
TypeScript estree import error

### ISSUE THREAD

**Issue by Leniorko** [prettier/prettier#15136]
**Environments:**

- Prettier Version: 3.0.0
- Running Prettier via: Browser API
- Runtime: Chrome V114
- Operating System: Linux
- Prettier plugins (if any): None? Probably.

**Steps to reproduce:**

I'm writing some application with svelte and TypeScript. And there is a component that is supposed to print code that it was passed from props. Code is usually not formatted so I'm using `Prettier` to format it at runtime and `highlight.js` to paint it.

It works perfectly fine. I imported firstly `babel` plugin and error in the browser said that I also need `estree`. I imported it and error disappeared and there is no more errors in browser. But typescript now throws an error about `estree` import: `File '.../node_modules/.pnpm/prettier@3.0.0/node_modules/prettier/plugins/estree.d.ts' is not a module.ts(2306)`

I suppose error happens because `estree.d.ts` is empty.

Just in case here is an entire code of the component:

```svelte
<script lang="ts">
  import hljs from "highlight.js";
  import type { LanguageFn } from "highlight.js";
  import type { Language } from "./CodeBlock.type";
  import { onMount } from "svelte";
  import "highlight.js/styles/stackoverflow-dark.css";
  import * as Prettier from "prettier";
  import BabelPlugin from "prettier/plugins/babel";
  import EsTreePlugin from "prettier/plugins/estree";

  export let codeBlock: string;
  let formattedCode: string = "";
  export let language: Language = "javascript";

  $: Prettier.format(codeBlock, {
    parser: "babel",
    plugins: [BabelPlugin, EsTreePlugin],
  }).then((result) => {
    formattedCode = result;
    console.log(result);
  });

  onMount(() => {
    hljs.registerLanguage(language, (hljs) => {
      return hljs.getLanguage(language) as ReturnType<LanguageFn>;
    });
  });

  let highlightedCode: string;

  $: highlightedCode = hljs.highlight(formattedCode, {
    language,
  }).value;
</script>

<code class="code hljs">
  {@html highlightedCode}
</code>

<style>
  .code {
    width: 100%;
    display: block;
    padding: 8px;
    white-space: pre-wrap;
  }
</style>
```

**Expected behavior:**

Proper import without any errors.

**Actual behavior:**

TypeScript error has type error, but works fine.

**Comment by auvred** [comment]
Hi, this issue is already fixed in https://github.com/prettier/prettier/pull/15018

**Comment by Leniorko** [comment]
> Hi, this issue is already fixed in #15018

Checked PR, looks like it. I'll be waiting for the next update then. Thank you.
